src/internal/tabbable.ts

```diff
@@ -23,7 +25,8 @@ function isTabbable(el: HTMLElement) {
   }
 
   // Elements that are hidden have no offsetParent and are not tabbable
-  if (el.offsetParent == null) {
+  // offsetParent polyfill is added because otherwise it misses elements in Safari.
```

Comment:
```suggestion
  // offsetParent() is added because otherwise it misses elements in Safari
```

[The spec was changed](https://github.com/w3c/csswg-drafts/issues/159), so it's not technically a polyfill. AFAIK there's no replacement for the previous behavior because it leaks shadow nodes.

Also, I'm curious why it only misses elements in Safari. Seems strange. 😕

Reply:
🤷 wish I knew.